configuration.yml

```diff
@@ -19,7 +19,7 @@ alluxio_port: "19998"
 pyspark_cassandra_version: "0.3.5"
 analysers_path: "/app/analysers"
 transformers_path: "/app/data-transformers"
-configurations_path: "/app/configuration"
+configurations_path: "/app/data-transformers/suts/wfms"
```

Comment:
`wfms` is a SUT type, and should be determined using the test configuration. The `configurations_path` I expect should be `/app/data-transformers/suts/`. Maybe a better name for the configuration variable would also help.